README.md

```diff
@@ -445,27 +445,27 @@ module.
 
 ### Markdown scripts
 
-The `zx` can execute scripts written in markdown 
+The `zx` can execute scripts embedded in markdown files
```

Comment:
Yes, the zx executes embedded code blocks. But let's think of this as a new syntax. Same as http://go/coffee-script#literate